src/structures/MessageEmbed.js

```diff
@@ -7,11 +7,19 @@ const Util = require('../util/Util');
  * Represents an embed in a message (image/video preview, rich embed, etc.)
  */
 class MessageEmbed {
-  constructor(data = {}) {
-    this.setup(data);
+  /**
+   * @param {Object} [data={}] The data to build the embed from
+   * @param {boolean} [skipValidation=false] A flag to skip field validations
```

Comment:
This parameter should be undocumented, since it's only for internal use.
(I don't think you can mark parameters as private, if that is possible, that would also be an option.)

Reply:
Should this param still be present in the typings, or removed there as well?

Reply:
It should be removed there too.

Reply:
Ah, if we remove one param eslint get's unhappy as the method doesn't have a correct docstring. I guess the best solution is just not to have a docstring at all?

Reply:
I'd say so.